server/index.js

```diff
@@ -1,7 +1,36 @@
-var express = require('express');
-var app = express();
-var port = 3004;
+const express = require('express');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+const morgan = require('morgan');
+// const router = require('./routes');
+const getData = require('../database/getData.js');
+const app = express();
+const port = 3004;
 
-app.use(express.static('public'));
+// app.use(express.static('public'));
+// app set at localhost:3004/listing?id=XXX
```

Comment:
I don't understand how this comment describes the following line.... line 11 seems to be a URL query param, while line 12 seems to be an entirely different endpoint